core/blockchain.go

```diff
@@ -1739,8 +1739,10 @@ func (bc *BlockChain) addFutureBlock(block *types.Block) error {
 //
 // After insertion is done, all accumulated events will be fired.
 func (bc *BlockChain) InsertChain(chain types.Blocks) (int, error) {
+	//log.Warn("QBFT: CALLING INSERT CHAIN CHAIN CHAIN", "len", len(chain))
```

Comment:
to remove debug logs? same in other places on the file

Reply:
Done